public/javascripts/livevalidation_standalone.js

```diff
@@ -898,4 +901,20 @@ var Validate = {
     	this.name = 'ValidationError';
     }
 
-}
\ No newline at end of file
+}
+
+function scrollToErrorIfFound() {
```

Comment:
I think we still need to move focus -- this behavior from the comments on Jira seems to still be present:

>It appears as though focus remains on the “Please wait…” button, so screen reader users aren’t taken to the field with the error. They’re basically still getting the same experience: the form is just mysteriously stuck.

Reply:
Sorry I missed that.

Reply:
That's okay! 

However, I'm testing the changes in Safari 15.5 on Mac OS 12.4, and it looks like putting focus on the field is now causing the error message to disappear, so that's something that probably needs a bit more work. Is it working okay in other browsers?

Reply:
The error message doesn't disappear after scrolling to it on Chrome and Firefox. 

But I've removed `removeMessageAndFieldClass()` entirely when live validation fields are in focus in [#4370](https://github.com/otwcode/otwarchive/pull/4379/files#diff-2b9e89120848a91c817a17dde80cf9ab19c41c8006ef12c121540c5f4bcd3c1dL213), so it might be fixed if you merge this with that PR.

Reply:
Yup, that takes care of it -- thanks! I'll make a note that this should be merged with or after #4370.